src/drivers/mct/main/seq_io_mod.F90

```diff
@@ -383,7 +383,7 @@ end function seq_io_sec2hms
   ! !INTERFACE: ------------------------------------------------------------------
 
   subroutine seq_io_write_av(filename,gsmap,AV,dname,whead,wdata,nx,ny,nt,fillval,pre,tavg,&
-       use_float, file_ind, scolumn)
+       use_float, file_ind, mask, scolumn)
```

Comment:
why did you put the new argument before an older one (scolumn) here?

Reply:
I should have been consistant and put them in the same order, but since they are both optional it doesn't matter.